costs/include/quadratic_cost.h

```diff
@@ -0,0 +1,85 @@
+#ifndef TORC_QUADRATIC_COST_H
+#define TORC_QUADRATIC_COST_H
+
+#include <string>
+#include "base_cost.h"
+
+namespace torc {
```

Comment:
Lets add a sub-namespace `cost` that goes in `torc`. It would be good to go back and add this to the linear cost class too. 

This will just help keep the name space less cluttered in the long run.